polynote-server/src/main/scala/polynote/server/repository/FileBasedRepository.scala

```diff
@@ -338,7 +338,7 @@ class FileBasedRepository(
     }
   }
 
-  override def createNotebook(relativePath: String, maybeContent: Option[String] = None): RIO[BaseEnv with GlobalEnv, String] = {
+  override def createNotebook(relativePath: String, maybeContent: Option[String] = None, maybeTemplate: Option[String] = None): RIO[BaseEnv with GlobalEnv, String] = {
```

Comment:
Is `maybeTemplate` the content of the template or the path to a template? Consider changing the name to reflect that (e.g., `maybeTemplatePath` or `maybeTemplateContent`)